forecast_script/entrypoint-scripts/01-extensions.sql

```diff
@@ -0,0 +1,2 @@
+create extension pg_cron;
+create extension plpython3u;
```

Comment:
Can be installed with env variables POSTGRES_MULTIPLE_EXTENSIONS=postgis,hstore

Reply:
ok, will try this.